.github/workflows/publish_v2_layer.yml

```
@@ -46,10 +47,12 @@ jobs:
           poetry export --format requirements.txt --output requirements.txt
           pip install -r requirements.txt
       - name: Set release notes tag
+        id: release-notes-tag
         run: |
           RELEASE_INPUT=${{ inputs.latest_published_version }}
           LATEST_TAG=$(git describe --tag --abbrev=0)
           RELEASE_TAG_VERSION=${RELEASE_INPUT:-$LATEST_TAG}
+          echo "::set-output name=release-tag-version::$RELEASE_TAG_VERSION"
```

```suggestion
          echo "release-tag-version=$RELEASE_TAG_VERSION" >> $GITHUB_OUTPUT
```

GitHub is going to deprecate `set-output`, let's start moving to the new format.

* **Docs**: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter
* **Deprecation**: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

---

Created maintenance task: https://github.com/awslabs/aws-lambda-powertools-python/issues/1591